src/services/expense.service.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+'use strict';
+
+let expenses = [];
```

Comment:
Why do we need let in this case?
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.

( I see reassign in code, but in this case you also can simply use const)